### PR TITLE
replace kVertex with kTag

### PR DESCRIPTION
**Pull request by cangfengzhs**
#### What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### What does this PR do?
Replace `KVertex` with `KTag_` to support vertex without tag

#### Which issue(s)/PR(s) this PR relates to?
https://github.com/vesoft-inc/nebula/issues/3123
  
#### Checklist：
- [ ] Documentation affected （Please add the label if documentation needs to be modified.)
- [ ] Incompatible （If it is incompatible, please describe it and add corresponding label.）
- [ ] Need to cherry-pick （If need to cherry-pick to some branches, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes：
Please confirm whether to reflect in release notes and how to describe:
>                                                                 `

### CODE REVIEW

**Review by critical27 — APPROVED** [review]

File: src/common/utils/Types.h [hunk 1]
```diff
@@ -12,12 +12,13 @@
 namespace nebula {
 
 enum class NebulaKeyType : uint32_t {
-  kVertex = 0x00000001,
+  kTag_ = 0x00000001,
```
Comment (Shylock-Hg) [inline src/common/utils/Types.h:15]: `kTag`?

Comment (cangfengzhs) [reply src/common/utils/Types.h:15]: `kTag` is defined in Base.h `constexpr char kTag[] = "_tag";`.And compiler will report a warning if a local variable overwrite global variables.

**Review by critical27 — APPROVED** [review]

File: src/common/utils/Types.h [hunk 2]
```diff
@@ -41,7 +42,7 @@ static typename std::enable_if<std::is_integral<T>::value, T>::type readInt(cons
 }
 
 // size of vertex key except vertexId
```
Comment (pengweisong) [inline src/common/utils/Types.h:44]: tag key

File: src/storage/CompactionFilter.h [hunk 3]
```diff
@@ -37,7 +37,7 @@ class StorageCompactionFilter final : public kvstore::KVFilter {
       return false;
     }
 
-    if (NebulaKeyUtils::isVertex(vIdLen_, key)) {
+    if (NebulaKeyUtils::isTag(vIdLen_, key)) {
       return !vertexValid(spaceId, key, val);
```
Comment (pengweisong) [inline src/storage/CompactionFilter.h:41]: should it be changed to `tagValid`?

**Review by pengweisong — APPROVED** [review]
change the comments mentioned, and others LGTM.

File: src/storage/CompactionFilter.h (hunk 3 above)
Comment (cangfengzhs) [reply src/storage/CompactionFilter.h:41]: Okey, I will fix it in next PR.

**Pull request merged**